site/content/en/docs/Reference/uninstalling_minikube.md

```diff
@@ -66,7 +66,7 @@ If you have installed minikube using the direct download method, follow the belo
 
 
 ## Brew
-If you have installed minikube using the direct download method, follow the below steps to uninstall minikube completely from your system -
+If you have installed minikube using [Homebrew](https://brew.sh/), follow the below steps to uninstall minikube completely from your system -
 - In the shell, type in `minikube delete` to delete the minikube cluster.
 - Uninstall the minikube package using `brew uninstall minikube`
 - Remove the directory containing the minikube configuration `rm -rf ~/.minikube`

```

Comment:
while we are at this, let remove this line. no need to delete minikube home folder manually.